### ISSUE TITLE
Working in terminal vim but not in GVim?

### ISSUE THREAD

**Issue by yhshin11** [junkblocker/unite-codesearch#3]
Hi there.

I'm having a bizarre issue where unite-codesearch is working in terminal vim but not in GVim. I am running Vim 7.4 on Ubuntu 14.04. I had a problem initially even with terminal vim, where it wasn't picking up your csearch fork, but I fixed that with the following:
`let g:unite_codesearch_command = "$HOME/gocode/bin/csearch -m %d '%s'"`

But for whatever reason, even this doesn't seem to work in GVim. Any ideas?

Edit:

After playing around a little bit, it seems that `let g:unite_codesearch_command = "$HOME/gocode/bin/csearch -m %d '%s'"` doesn't actually get picked up even in terminal vim, and it's still using `'csearch -m %d '%s'"` for whatever reason. If I run terminal vim, it still picks up the correct executable since I have the correct PATH variable set in my .bashrc file, whereas gVim doesn't execute my .bashrc file so it doesn't pick it up correctly.

So basically it seems like the problem is that the g:unite_codesearch_command option isn't getting set properly.
